client/src/pages/EditAccount.js

```diff
@@ -2,22 +2,30 @@ import React from "react";
 import { useForm, Controller } from "react-hook-form";
 import { connect } from "react-redux";
 import Checkbox from "components/Input/Checkbox";
-import SubmitButton from "components/Button/SubmitButton";
-import styled from "styled-components";
+import { getInitials } from "utils/userInfo";
 import FormInput from "components/Input/FormInput";
-import Heading from "components/Typography/Heading";
+import ProfilePic from "components/Picture/ProfilePic";
+import { Link } from "react-router-dom";
 import UnderLineDescription from "components/Input/UnderlineDescription";
-import { theme } from "constants/theme";
-
-const { royalBlue } = theme.colors;
-
-const Label = styled.label`
-  color: ${(props) => props.inputColor || `${royalBlue}`};
-  padding-left: ${(props) => props.paddingLeft || ""};
-  margin-top: ${(props) => props.marginTop || "1.5rem"};
-  font-size: ${(props) => props.size || ""};
-  font-weight: ${(props) => props.weight || ""};
-`;
+import locationIcon from "assets/icons/location.svg";
```

Comment:
I wonder if it's the same icon that I added on PR for edit-profile

Reply:
if its the cone looking blue icon in front of location then yes

Reply:
yeah I guess, merged that other PR so maybe you can check if its the same and doesnt have merge conflict? I also think probably we want to update the svg's later so we can add color as a prop instead of hardcoding it inside the svg